mod/authors.txt

```diff
@@ -1,23 +0,0 @@
-Author:

```

Comment:
Why delete this? I don't see a reason to not keep this.

Reply:
This file was in two places. The root of the repository and inside `mod`. I am removing the duplicate from mod.